### ISSUE TITLE
UIMaker Slideout Error on Reloads

### ISSUE THREAD

**Issue by mda-hdm** [eirannejad/pyRevit#1080]
Hi there,

I'm experiencing a small issue when triggering reloads in a script that actually tries to pull some extension repositories.
The error says that UIMaker can't create a slideout since it already exists. As far as I can see there is not real problem and everythings works fine. The only problem is that a lot of my colleagues that use pyRevit kind of get confused by that message. 

My guess is that the `ext_asm_info` might not be having the correct value for the `reloading` property.

https://github.com/eirannejad/pyRevit/blob/92cb346eece706db810468984b3fa8564f14b16c/pyrevitlib/pyrevit/loader/uimaker.py#L158

Is there anything I can do from my side to avoid that error when triggering reloads?

**Comment by eirannejad** [comment]
Great. I'll improve the logging and possible will get rid of the error

**Comment by mda-hdm** [comment]
Thank you very much!

**Comment by eirannejad** [comment]
I looked at the code and it seems to be fine as long as pyRevit knows it is being reloaded. Are you using `sessionmgr.reload_pyrevit()` to reload pyRevit?

**Comment by mda-hdm** [comment]
Yes exactly. I use basically the same as you do in the default reload button. It somehow also only affect slideouts. Is there any other way to trigger reloads? 

Thank you very much for the great work!

**Comment by eirannejad** [comment]
Not really. Let me try running this from a script and see what I can do

**Comment by mda-hdm** [comment]
The pyRevit reloading happens [in this method](https://github.com/revitron/revitron-ui/blob/94e69fa4aae39eb7fb58025e18dfd77bca0ff7f1/lib/rpm/system/session.py#L8). 

Originally updates have been issued automatically on Revit startup. I figured out that _this could be the issue_ since the reload was triggered while the startup process was still running. I removed the startup checks for now and the error is gone so far.
Also note that apparently that error had no impact to the functionality. So it should be fine. 😄
